src/wallet/test/coinselector_tests.cpp

```diff
@@ -136,6 +136,14 @@ inline std::vector<OutputGroup>& GroupCoins(const std::vector<COutput>& coins)
     return static_groups;
 }
 
+inline std::vector<OutputGroup>& KnapsackGroupOutputs(const CoinEligibilityFilter& filter)
```

Comment:
Nit: the introduction `KnapsackGroupOutputs` should be mentioned in the commit message. Should be potentially a separate commit.

Reply:
I've mentioned it in the commit message